src/main/java/org/graylog/plugins/threatintel/functions/tor/TorExitNodeLookupFunction.java

```diff
@@ -42,12 +43,11 @@ public GenericLookupResult evaluate(FunctionArgs args, EvaluationContext context
 
         final LookupResult lookupResult = this.lookupFunction.lookup(ip.trim());
         if (lookupResult != null && !lookupResult.isEmpty()) {
+
+            // If not a String, then fall through to false at the end of the method.
             final Object value = lookupResult.singleValue();
-            if (value instanceof Boolean) {
-                return (Boolean) value ? GenericLookupResult.TRUE : GenericLookupResult.FALSE;
-            }
             if (value instanceof String) {
-                return Boolean.valueOf((String) value) ? GenericLookupResult.TRUE : GenericLookupResult.FALSE;
+                return Strings.isNotBlank((String) value) ? GenericLookupResult.TRUE : GenericLookupResult.FALSE;
```

Comment:
We usually use `com.google.common.base.Strings.isNullOrEmpty` in such cases. I think this is a better choice here so we don't pull in a class from the specific logger implementation.